### ISSUE TITLE
0 Iterations

### ISSUE THREAD

**Issue by menco22** [chkwon/PyHygese#26]
Whenever the algorithm is used via wrapper (including test examples, both with the distance matrix and with the coordinates), I always get 0 iterations as a result and a total time for the algorithm's execution that does not conform to the times given to me by the HGS in C++. Furthermore, intermediate iterations are not printed (probably related to the initial problem of 0 iterations).
![Immagine 2024-04-10 190317](https://github.com/chkwon/PyHygese/assets/114168420/96b0ac91-9b87-4d0c-9e22-72964f3665b0)

**Comment by menco22** [comment]
solved!

**Comment by chkwon** [comment]
Sounds great. What was the issue?

**Comment by menco22** [comment]
I apologize for the delayed response, but university commitments have fully occupied my time, and consequently, I had to set aside this project. The problem was due to a previous incorrect installation of HYGESE in my project. I take this opportunity to ask another question regarding the code: is the distance matrix that we pass as input to the algorithm made up of spatial or temporal distances? Because looking at the Individual.cpp section, it seems that the distances are actually temporal! I apologize again for the delayed response and thank you in advance for your attention.
